flocker/control/script.py

```diff
@@ -0,0 +1,42 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Script for starting control service server.
+"""
+
+from twisted.python.usage import Options
+from twisted.internet.endpoints import TCP4ServerEndpoint
+
+from .httpapi import create_api_service
+from ..common.script import (
+    flocker_standard_options, FlockerScriptRunner, main_for_service)
+
+
+@flocker_standard_options
+class ControlOptions(Options):
+    """
+    Command line options for ``flocker-control`` cluster management process.
+    """
+    optParameters = [
+        # XXX also choose where configuration will be stored
```

Comment:
No, this isn't a configuration file. This is data *written* by the server. So it'd go in `/var/lib/flocker` or something. The word "configuration" is a bit misleading, it's configuration of the cluster, not of the control service process as stuff in `/etc/` would be.